kafka-connect-cassandra/src/main/scala/com/datamountaineer/streamreactor/connect/cassandra/config/CassandraSettings.scala

```diff
@@ -134,20 +125,9 @@ object CassandraSettings extends StrictLogging  {
     val fields = config.getFields(routes)
     val ignoreFields = config.getIgnoreFields(routes)
 
-    val threadPoolSize: Int = {
-      val threads = config.getInt(CassandraConfigConstants.SINK_THREAD_POOL_CONFIG)
-      if (threads <= 0) 4 * Runtime.getRuntime.availableProcessors()
-      else threads
-    }
+    val threadPoolSize: Int = config.getThreadPoolSize
 
-    val consistencyLevel = config.getString(CassandraConfigConstants.CONSISTENCY_LEVEL_CONFIG) match {
-      case "" => None
-      case other =>
-        Try(ConsistencyLevel.valueOf(other)) match {
-          case Failure(e) => throw new ConfigException(s"'$other' is not a valid ${CassandraConfigConstants.CONSISTENCY_LEVEL_CONFIG}. Available values are:${ConsistencyLevel.values().map(_.name()).mkString(",")}")
-          case Success(cl) => Some(cl)
-        }
-    }
+    val consistencyLevel = config.getConsistencyLevel(classOf[ConsistencyLevel])
```

Comment:
This deserves a comment to explain why you are passing the classOf instead of a variable